README.md

```diff
@@ -59,6 +59,8 @@ Finally, navigate to [app.plex.tv/desktop](https://app.plex.tv/desktop) or [your
 - `$ADVERTISE_IP`         - This variable defines the additional IPs on which the server may be be found. For example: `http://10.1.1.23:32400`. This adds to the list where the server advertises that it can be found.
 - `$DISABLE_REMOTE_SEC`   - 
 - `$PLEX_CLAIM`           - The claim token for the server to obtain a real server token. If not provided, server will not be automatically logged in. If server is already logged in, this parameter is ignored.
+- `$LOGDEBUG`             - Disables debug logging if set to 0, and defaults to 1 (enables debug logging).
```

Comment:
These need to be updated to have the underscore in them like the code now does. Could there also be a note somewhere in the readme saying that these will override the Plex UI options for debug and verbose.

Reply:
@qdm12

Reply:
Hi there, sorry for forgetting to update the doc. I'll do it today/tomorrow. Thanks for letting me know!